src/bp/ui-studio/src/web/views/FlowBuilder/sidePanel/index.tsx

```diff
@@ -1,14 +1,19 @@
 import { Icon } from '@blueprintjs/core'
 import _ from 'lodash'
 import reject from 'lodash/reject'
+import values from 'lodash/values'
 import React, { Component } from 'react'
+import { connect } from 'react-redux'
 import { RouteComponentProps } from 'react-router'
+import { deleteFlow, duplicateFlow, renameFlow } from '~/actions'
 import { SidePanel, SidePanelSection } from '~/components/Shared/Interface'
+import { getCurrentFlow, getDirtyFlows } from '~/reducers'
+
+import Inspector from '../inspector'
```

Comment:
Eventually i'd like to fix the casing of components on the studio. Having a file index.tsx in a folder named "inspector" instead of "Inspector" mess up the naming.

It also gives warnings with the ts flag  forceConsistentCasingInFileNames), and in vscode i've encountered a bug where sometime you can have the file "sidePanel/FlowTools.tsx" and "SidePanel/FlowTools.tsx" to be opened at the same time